mentat/code_feature.py

```diff
@@ -132,6 +130,10 @@ def __init__(
                     raise MentatError("CodeFeatures should only have on interval.")
                 self.interval = interval[0]
                 level = CodeMessageLevel.INTERVAL
+
+        if not self.path.is_absolute():
```

Comment:
I think it might be good to name this self.abs_path (just to make extra clear that it's always absolute); not entirely necessary if you don't want to though.

Reply:
My opinion is that all paths should be assumed to be absolute unless specified otherwise. Like we have `rel_path` and `posix_path` variables for non-absolute paths which I think we should continue doing.